apps/server/src/shared/infra/collaborative-storage/collaborative-storage.adapter.spec.ts

```diff
@@ -75,11 +89,27 @@ describe('CollaborativeStorage Adapter', () => {
 		});
 	});
 
-	describe('Delete Team from Nextcloud', () => {
-		it('should call the strategy', () => {
-			const teamIdMock = 'teamIdMock';
-			adapter.deleteTeam(teamIdMock);
-			expect(strat.deleteGroupfolderAndRemoveGroup).toHaveBeenCalled();
+	describe('deleteTeam', () => {
+		it('should call the strategy', async () => {
+			const teamIdMock = new ObjectId().toHexString();
+			await adapter.deleteTeam(teamIdMock);
+			expect(strat.deleteTeam).toHaveBeenCalled();
```

Comment:
could you use .toHaveBeenCalledWith(teamIdMock) as you have in other unit tests?

Reply:
For sure. Done :)